munch-match/src/App.css

```diff
@@ -62,31 +62,7 @@ button {
   z-index: 1;
 }
 
-.button {
-  background-color: #dbdeff;
-  border: 8px solid white;
-  border-radius: 25px;
-  margin: 0.5%;
-  z-index: 2;
-}
-
-.results {
-  padding: 1.5%;
-  width: 33.33%;
-  flex-grow: 1;
-  display: flex;
-  flex-direction: column;
-  justify-content: space-between;
-}
-
-.food {
-  width: 160px;
-  height: 160px;
-  object-fit: cover;
-  overflow: hidden;
-}
-
-.surpise {
+.surprise {
```

Comment:
ik you didn't add this class but it's unused so can we remove it hehe

Reply:
dam also all anything with `App-logo`

Reply:
Yep will remove those